css/base/base.css

```diff
@@ -0,0 +1,20 @@
+@import "reset.css";
+
+/* global variables */
+:root{
+    --primary-color: #E40C0B;
```

Comment:
Try to format the code using prettier.

Reply:
Hey Tahir, done the formatting please take a look.